contracts/examples/ping-pong-egld/mandos/ping-pong-call-ping-before-beginning.scen copy.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "try to ping before the contract has begun",
+    "steps": [
+        {
+            "step": "externalSteps",
+            "path": "ping-pong-init.scen.json"
+        },
+        {
+            "step": "setState",
+            "accounts": {
+                "address:participant1": {
```

Comment:
I usually initialize all participants in a steps.json and then just import that in all tests before everything.

Reply:
Agreed. Done.